resources/views/upcoming.blade.php

```diff
@@ -0,0 +1,188 @@
+@extends('layout')
+
+@section('title', 'Calendar')
+
+@section('content')
+
+@component('sidebar', ['selected' => 'calendar'])
+@endcomponent
+
+<div class="main">
+
+  @component('header')
+    @slot('title') Schedule @endslot
+      <div class="header-tab-item tab-selected">Upcoming</div>
+      <a href="/calendar" style="text-decoration:none;"><div class="header-tab-item">Calendar</div><a>
+   @endcomponent
+
+        <div class="width-scrollable">
+
+        <div id="upcoming-widget">
+        </div>
+
+          <!--<div class="column">
```

Comment:
Need to delete all these commented lines of code

Reply:
Ok. Thanks for your suggestion!